com.unity.multiplayer.mlapi/Runtime/Core/NetworkedBehaviour.cs

```diff
@@ -48,13 +48,14 @@ internal enum NExec
         // RuntimeAccessModifiersILPP will make this `protected`
         internal BitWriter BeginSendServerRpc(ServerRpcSendParams sendParams, bool isReliable)
         {
-            var rpcQueueMananger = NetworkingManager.Singleton.RpcQueueManager;
+            NetworkingManager networkingMgr = NetworkingManager.Singleton;
```

Comment:
nit: I personally prefer to keep stuff like this to a separate PR. There are 58 changes to this file but only ~3 look relevant?